src/App.jsx

```diff
@@ -24,16 +24,14 @@ const Layout = ({
     appsInfoQuery,
 }) => {
     return (
-        <>
+        <div className={styles.container}>
             <ConnectedHeaderBar
                 clientPWAUpdateAvailable={clientPWAUpdateAvailable}
                 onApplyClientUpdate={onApplyClientUpdate}
                 appsInfoQuery={appsInfoQuery}
             />
-            <div className={styles.container}>
-                <Outlet />
-            </div>
-        </>
+            <Outlet />
+        </div>
```

Comment:
Part of a fix to have plugin size be (page height - header bar height), instead of just page height (which made it too tall)